sprayjson/src/main/scala/io/finch/sprayjson/package.scala

```diff
@@ -0,0 +1,23 @@
+package io.finch
+
+import com.twitter.io.Buf
+import com.twitter.util.Try
+import spray.json._
+
+package object sprayjson{
+
+  /**
+    * @param format spray-json support for convert JSON val to specific type object
+    * @tparam A the type of the data to decode into
+    */
+  implicit def decodeSpray[A](implicit format:JsonFormat[A]): Decode[A] = Decode.instance[A]({
```

Comment:
Please, add space after `:`.

Also, you don't need to wrap closure with `{}`. Just `Decode.instance(s => Try(...))` works great.